Coding-Standards.md

```diff
@@ -1,6 +1,7 @@
 # Chef Companion Coding Standards
 This document will contain our agreed upon coding standards for development.
-
+## JavaScript Code Format
+All code in this language should be run through Prettier for consistent code formatting.
```

Comment:
With a small project like this, restrictions will be rather relaxed. Prettier and ESLint on their default settings will be used to enforce naming conventions, format, and structure for JavaScript. There will be no checks for python, however [PEP8](https://peps.python.org/pep-0008/) will be expected.